### DISCUSSION TITLE
TypeSpec postgres emitter

### DISCUSSION THREAD

**Discussion by NielsCo** [microsoft/typespec#2115] [Show and tell]
Hey everyone,
I just released my typespec-postgres emitter as an [open-source project](https://github.com/NielsCo/typespec-postgres)
Any feedback is welcome! It's also available as an npm package ```npm i typespec-postgres```

It's still in the early stages in development and there are a lot of things that need to be ironed out.

**Comment by bterlson** [comment]
This looks awesome!  😁
